test/video/autoplay.js

```diff
@@ -21,14 +21,14 @@ Checks for support of the autoplay attribute of the video element.
 
   Modernizr.addAsyncTest(function() {
     var timeout;
-    var waitTime = 500;
+    var waitTime = 300;
     var elem = createElement('video');
     var elemStyle = elem.style;
 
     function testAutoplay(arg) {
       clearTimeout(timeout);
       elem.removeEventListener('playing', testAutoplay, false);
-      addTest('videoautoplay', arg && arg.type === 'playing' || elem.currentTime !== 0 || !elem.paused);
+      addTest('videoautoplay', arg && arg.type === 'playing' || !elem.paused || elem.currentTime !== 0);
```

Comment:
this change is actually not relevant in this case, but it would be against modernizr's master